src/raftstore/store/peer.rs

```diff
@@ -2153,7 +2196,7 @@ mod tests {
             lease_state: LeaseState,
         }
         impl RequestInspector for DummyInspector {
-            fn has_applied_to_current_term(&self) -> bool {
+            fn has_applied_to_current_term(&mut self) -> bool {
```

Comment:
Why does this now need to be mutable?

Reply:
I want to record requsts that be rejected by term mismatch, it needs to be mutable.